jans-cedarling/bindings/cedarling_c/tests/test_cedarling.c

```
@@ -667,28 +667,237 @@ void test_trusted_issuer_loading_info() {
+    cedarling_free_instance_result(&instance_result);
+
+    // We only exercise the FFI boundary here: send an empty-tokens payload
+    // and expect a validate-time rejection. Full happy-path multi-issuer
+    // decisions require a JWT-generating harness (not present in the C
+    // test fixture); core lib + WASM/Python/UniFFI/Java suites cover that.
+    const char* empty_tokens =
+        "{\"tokens\": [], \"items\": [{\"resource\": {\"cedar_entity_mapping\": {\"entity_type\": \"Jans::Issue\", \"id\": \"a\"}}, \"action\": \"Jans::Action::\\\"Update\\\"\", \"context\": {}}]}";
+
+    CedarlingResult auth_result;
+    ret = cedarling_authorize_multi_issuer_batch(instance_id, empty_tokens, &auth_result);
+    TEST_ASSERT(ret != 0, "Empty tokens batch is rejected");
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)